lib/ansible/modules/database/postgresql/postgresql_ext.py

```diff
@@ -105,7 +105,7 @@
 - The default authentication assumes that you are either logging in as
   or sudo'ing to the C(postgres) account on the host.
 - This module uses I(psycopg2), a Python PostgreSQL database adapter.
-- You must ensure that psycopg2 is installed on the host before using this module.
+- You must ensure that I(psycopg2) is installed on the host before using this module.
```

Comment:
```suggestion
- You must ensure that C(psycopg2) is installed on the host before using this module.
```
I'm not 100% sure but it seems that we usually use C(..) to highlight such things. @felixfontein ? ^

Reply:
Yes, it's usually being done in the cases I know of.